### PR TITLE
Add admin methods for reindexing versions from project and version admin.

### PR DESCRIPTION
**Pull request by dojutsu-user**
Related issue - #4654 

- [x] On Project admin, a command that reindexes all active versions (project queryset and all attached active versions)
- [x] On Version admin, a command that reindexes by version queryset instead
- [ ] ~Drop management command, these aren't good interfaces for site admins and are mostly hidden~
- [x] The management command seems to just do all versions active or not, we should probably test for active versions instead.
- [x] There should also be an admin function to wipe the index for a version and for a project

### CODE REVIEW

**Comment by dojutsu-user** [comment]
@ericholscher 
> You should be able to call https://github.com/rtfd/readthedocs.org/blob/master/readthedocs/search/signals.py#L44

([comment #1 ](https://github.com/rtfd/readthedocs.org/pull/4992#issuecomment-458726388))
Calling `index_project_save` will not do anything because `ELASTICSEARCH_DSL_AUTO_REFRESH` is set to `False`

Same reason for not calling `index_html_file` directly.

**Comment by dojutsu-user** [comment]
I apologise that this PR took too much time from my side.

**Comment by ericholscher** [comment]
@dojutsu-user did you test this locally? I'm seeing it reindexing _all_ projects.

File: readthedocs/projects/admin.py [hunk 1]
```diff
@@ -222,6 +229,56 @@ def build_default_version(self, request, queryset):
...
+            if not active_versions.exists():
+                self.message_user(
+                    request,
+                    'No active versions of project {}'.format(project),
+                    messages.ERROR
+                )
+            else:
+                for version in active_versions.iterator():
+                    html_objs = HTMLFile.objects.filter(project=project, version=version)
+                    _indexing_helper(html_objs, wipe=False)
```
Comment (ericholscher) [inline readthedocs/projects/admin.py]: This is reindexing a number of times, it should only run once.

File: readthedocs/search/utils.py [hunk 2]
```diff
@@ -67,3 +68,25 @@ def _get_document(model, document_class):
...
+
+
+def _indexing_helper(html_objs, wipe=False):
+    """
+    Helper function for reindexing and wiping indexes of projects and versions.
+
+    If ``wipe`` is set to False, html_objs are deleted from the ES index,
+    else, html_objs are indexed.
+    """
+    from readthedocs.search.tasks import index_objects_to_es, delete_objects_in_es
```
Comment (ericholscher) [inline readthedocs/search/utils.py]: This should check if `html_objs` exists -- it might not.

**Comment by dojutsu-user** [comment]
@ericholscher 

> did you test this locally

I did tested this locally.

> It looks like this happens when there is a version with no `html_objs`, it continues to pass it on to ES which indexes everything.

I have added multiple `if` to prevent this from happening.

File: readthedocs/projects/admin.py (hunk 1 above)
Comment (dojutsu-user) [reply readthedocs/projects/admin.py]: Thank you.
I have made and pushed the changes.

File: readthedocs/search/utils.py (hunk 2 above)
Comment (dojutsu-user) [reply readthedocs/search/utils.py]: I have added the checks to prevent this.

**Pull request merged**